UPGRADE_GUIDE.md

```diff
@@ -1,5 +1,34 @@
 # Upgrade Guide
 
+## From `0.3.3` to `0.3.4`
+
+### PlayerLifecycle feature module now provides an EntityId
+
+The callback used for creating a player `EntityTemplate` has changed to provide an `EntityId` up front.
+This will be the `EntityId` the player's Entity will have after it is successfully spawned, and can be usefull for QBI.
+
+```csharp
+// Previously
+public static EntityTemplate Player(string workerId, byte[] args)
+{
+    var template = new EntityTemplate();
+    // ...
+    return template;
+}
+
+PlayerLifecycleConfig.CreatePlayerEntityTemplate = FpsEntityTemplates.Player;
```

Comment:
This a little confusing since this won't compile if you paste it somewhere and typically lives in another file

Reply:
I can leave it out if you'd like?

Reply:
Yeah I think that's preferable. It doesn't change during the upgrade as well